### ISSUE TITLE
Add Tx Fees included option in Make Payment feature

### ISSUE THREAD

**Issue by yorotsuya** [yorotsuya/otk-poc#23]
Usually, Tx fees is exclude from payment, however, the request option should also be available by users demand.

**Comment by jengjy** [comment]
Does it mean when the Tx fees is included, if I send 10000 satoshis and the tx fees is 1000 satoshis  then, recipient can only get 9000 satoshis?

**Comment by yorotsuya** [comment]
Yes, but this is a secondary feature. Firstly we want to let users being able to configure the TX fee.

從 ProtonMail 手機版送出

--------原始郵件--------
開啟 2019年6月3日 上午10:55, notifications@github.com 寫道：
